packages/open-next/src/build/patch/patches/patchEnvVar.ts

```diff
@@ -0,0 +1,46 @@
+import { createPatchCode } from "../astCodePatcher.js";
+import type { CodePatcher } from "../codePatcher";
+
+export const envVarRuleCreator = (envVar: string, value: string) => `
+rule:
+  kind: member_expression
+  pattern: process.env.${envVar}
+  inside:
+    kind: if_statement
```

Comment:
`if_statement` has both the condition and the block statement, I don't think that is what we are looking for here?

Maybe also add a comment on the functionμ?